src/aws-api-mcp-server/tests/aws/test_service.py

```diff
@@ -533,3 +536,41 @@ def test_profile_not_added_if_present_for_customizations(mock_get_region, mock_m
     assert '--profile' in args
     profile_index = args.index('--profile')
     assert args[profile_index + 1] == 'different'
+
+
+@pytest.mark.parametrize(
```

Comment:
All these tests cover file in the current directory, do we support relative/absolute paths as well?